src/classes/BDI_DataImportService.cls

```diff
@@ -721,9 +721,45 @@ global with sharing class BDI_DataImportService {
                 }
             }
         }
+        if (isCurrencyMapped(strBaseObj)) {
+            Boolean isCurrencyIsoCodeMapped = false;
```

Comment:
@mdeleon0 Can you say more about what this block is doing?

Reply:
Sure @lmeerkatz I will add a comment.